packages/pds/src/services/moderation/views.ts

```diff
@@ -179,8 +181,7 @@ export class ModerationViews {
           'in',
           results.map((r) => didFromUri(r.uri)),
         )
-        .selectAll('repo_root')
-        .selectAll('did_handle')
+        .selectAll(['repo_root', 'did_handle'])
```

Comment:
so, the idea here is to only expose this field through `admin.getRepo()` endpoint and the lexicon property `com.atproto.admin.defs#repoViewDetail`. the primary use-case for this right now is to just show email verified accounts with a special flag on ozone's repository view page so trying to avoid exposing this on every `repoView` slice.
which is also why the field itself is defined as optional since the only time we're actually fetching it from the db before passing it along to the view is from the `account().getAccount()` [call](https://github.com/bluesky-social/atproto/blob/main/packages/pds/src/api/com/atproto/admin/getRepo.ts#L13)

LMK if that tracks.

Reply:
Ahh, I am getting it now, thanks!

Since `emailConfirmedAt` is part of `RepoViewDetail` but not `RepoView`, I think it would be sort of nice/tidy for that difference to be reflected in `repoDetail()` the same way the other differences between those two views are.  With this change there's a bit of coupling between how the result set is generated versus whether it's suitable to be passed to `repo()` versus `repoDetail()`, and I wonder if it might be a little simpler to always select `emailConfirmedAt` then place the logic to include the field in the view inside `repoDetail()`.

All that said this is quite a small thing, I certainly wouldn't block on this 👍